shards/modules/inputs/inputs.cpp

```diff
@@ -536,39 +513,45 @@ struct HandleURL : public Base {
   static SHTypeInfo outputType() { return CoreInfo::AnyType; }
 
   SHVar activate(SHContext *context, const SHVar &input) {
-    for (auto &ev : _inputContext->events) {
-      if (ev.type == SDL_DROPFILE || ev.type == SDL_DROPTEXT) {
-        _url.clear();
-        auto url = ev.drop.file;
-        _url.append(url);
-        SDL_free(url);
+    // TODO: Input
```

Comment:
Hmm we need this on iOS cos it's the same thing to transfer things between apps

Reply:
#731